src/main/java/calculator/CalculatorMain.java

```diff
@@ -0,0 +1,21 @@
+package calculator;
+
+public class CalculatorMain {
+    public static void main(String[] args) {
+        String inputFilePath = "src/main/resources/input.txt";
+        String outputFilePath = "src/main/resources/output.txt";
+
+        GetDataFromInputFile getDataFromInputFile = new GetDataFromInputFile(inputFilePath);
```

Comment:
here could be problems with supporting this code when we decide to change getting data from DB for example or even have both of them - from file and from DB.
Please take some time to read about the Dependency Injection concept.

Reply:
Dependency Injection concept was added to QA_plan_for_interns document for future implementation into the project.